### PR TITLE
dbeaver/pro#1319 check password expiration; fix password changing

### PR DESCRIPTION
**Pull request by LonwoLonwo**
2023-01-03 14_20_43-DBeaver Ultimate 22 3 1 - EmpView

Please check:
- password changing with https://github.com/dbeaver/dbeaver/wiki/Change-current-user-password
- password expiration message (one per connection)

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ext.vertica/src/org/jkiss/dbeaver/ext/vertica/internal/VerticaMessages.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    static final String BUNDLE_NAME = "org.jkiss.dbeaver.ext.vertica.internal.VerticaMessages"; //$NON-NLS-1$
+
+    static {
+        // initialize resource bundle
+        NLS.initializeMessages(BUNDLE_NAME, VerticaMessages.class);
+    }
+
+    public VerticaMessages() {
+    }
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ext.vertica/src/org/jkiss/dbeaver/ext/vertica/internal/VerticaMessages.java]:
```suggestion
    private VerticaMessages() {
        // prevents construction
    }
```

File: plugins/org.jkiss.dbeaver.ext.vertica/src/org/jkiss/dbeaver/ext/vertica/internal/VerticaMessages.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+vertica_password_will_expire_warn_name = Password will be expired soon
+vertica_password_will_expire_warn_description = Details: {0}\n\nTo change your current password, click the Navigator tree on your connection and choose the Security section -> Change user password. Or contact your DBA.
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ext.vertica/src/org/jkiss/dbeaver/ext/vertica/internal/VerticaMessages.properties]:
```suggestion
vertica_password_will_expire_warn_description = Details: {0}\n\nTo change your current password, open the context menu on your database connection in the navigator, choose "Security" -> "Change user password".\nIf you're unable to change the password, contact your DBA.
```

File: plugins/org.jkiss.dbeaver.ext.vertica/src/org/jkiss/dbeaver/ext/vertica/internal/VerticaMessages.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+public class VerticaMessages extends NLS {
+
+    static final String BUNDLE_NAME = "org.jkiss.dbeaver.ext.vertica.internal.VerticaMessages"; //$NON-NLS-1$
+
+    static {
+        // initialize resource bundle
+        NLS.initializeMessages(BUNDLE_NAME, VerticaMessages.class);
+    }
+
+    public VerticaMessages() {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ext.vertica/src/org/jkiss/dbeaver/ext/vertica/internal/VerticaMessages.java]:
```suggestion
    private VerticaMessages() {
```

**Review by E1izabeth — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Comment by uslss** [comment]
verfified

**Pull request merged**